pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroup.java

```diff
@@ -128,7 +130,7 @@ public ResourceGroup(ResourceGroup other) {
         }
     }
 
-    protected void updateResourceGroup(ResourceGroupConfigInfo rgConfig) {
+    protected void updateResourceGroup(org.apache.pulsar.common.policies.data.ResourceGroup rgConfig) {
```

Comment:
Why change to include the full package name here?

Reply:
the configuration datastructure (visible to user) and the internal datastructure are both called ResourceGroup. So, fully qualifying name was needed to dis-ambiguate.